.size-limit.js

```diff
@@ -1,7 +1,7 @@
 module.exports = [
   {
     path: 'lib/index.js',
-    limit: '200 KB',
+    limit: '250 KB',
```

Comment:
Had to increase the limit as the upgrade added some additional overhead. We're at about 238K now.